frontend/device/remarkable/event_map.lua

```diff
@@ -0,0 +1,7 @@
+return {
+    [102] = "Home",
+    [105] = "LPgBack",
+    [106] = "RPgFwd",
+    [116] = "Power",
+}
+-- TODO libremarkable has 143 as "wakeup" - don't know what that corresponds to?
```

Comment:
Does it happen to have a hall effect sensor? In which case, my guess would be the key event that's sent by that when the magnet is lifted off ;).

Reply:
Ah possibly, I'll try running a magnet around the device and see if I get any events. I don't have the remarkable case so I'd not thought about that possibility.

Reply:
I did try running a magnet around the bezels but I couldn't get any events out of it